### PR TITLE
Project 3: Using Bootstrap - Milestone 3

### PR DESCRIPTION
**Pull request by arthurborgesdev**
Add the following sections to the project: 

- In the magazine 
- Editor’s pick
- Newspaper main sections:
  - US
  - World
  - Business
  - Tech & Science
  - Culture
  - Sports
  - Expert forum

Prepare to work on "Subscribe / Newsletter"

Adjust README.md to relate to the whole project description

### CODE REVIEW

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__You've done well so far in this milestone. I must acknowledge you for using a very small amount of vanilla CSS rules :handshake: 
However, some changes are required before it can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

_**Remember that you are expected to be more precise and to make sure that your page looks exactly the same as the source page.**_
- [x] Your live demo seems to be broken. Kindly fix it.
- [x] Ensure that the navbar is fixed to the top of the page when scrolling.
- [x] In the magazine and Editor's pick
- Kindly bold each of the title of the articles
- You should reduce the spaces (margin/padding) to the left and right, for mobile screen sizes. Check the image below:

| Original | Your clone |
| ---------- | --------------- |
| original | clone |

- [x] Main newspaper sections (US, World, etc.) Section
- Each title (U.S., World, etc) uses a hx tag and should **have a link**
- Do not forget to bold each of the titles of the articles

- [x] The Subscribe / Newsletter section is part of milestone3. Please ensure that you add this section and make all required changes before requesting another review.



* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

File: index.html
Comment (bini-i) [inline index.html]: - [x] Please make sure input element has type of email

**Review by bini-i — CHANGES_REQUESTED** [review]
# Status: Changes requred :yellow_circle:  
### Hello  team :wave: 

#### Thanks for working on the requirements suggested by the previous TSE but you should work on all requirements for this project to be approved. Please work on the following requirements and submit them for review:

- [x] Newsletter section has an input with type **email** (refer inline comment below)
- [x] Inside Footer, edition section is overflowing and causing a horizontal scroll, please fix the overflow
> Original 
image
> Clone 
image

- [x] Include screenshot in your README

#### If you have any question kindly reach me on slack @Binyam Hailemeskel

### HAPPY CODING :computer: :blush: 

> #### Reviewed by Binyam Hailemeskel (TSE)
> GitHub: [@bini-i](https://github.com/bini-i)
> Twitter: [@binyamshewa](https://twitter.com/binyamshewa)
> LinkedIn: [LinkedIn](https://www.linkedin.com/in/binyam-hailemeskel-728048151/)

**Review by bini-i — APPROVED** [review]
# Status: Approved :heavy_check_mark: : 
### Hello  team :wave: 

#### Congratulations :tada: , you have met all the requirements and your project is approved. 

#### If you have any question kindly reach me on slack @Binyam Hailemeskel

### HAPPY CODING :computer: :blush: 

> #### Reviewed by Binyam Hailemeskel (TSE)
> GitHub: [@bini-i](https://github.com/bini-i)
> Twitter: [@binyamshewa](https://twitter.com/binyamshewa)
> LinkedIn: [LinkedIn](https://www.linkedin.com/in/binyam-hailemeskel-728048151/)

**Pull request merged**
